crt_portal/cts_forms/templates/forms/snippets/pagination.html

```diff
@@ -0,0 +1,38 @@
+<nav class="usa-pagination" role="navigation" aria-label="Pagination">
+  <ol class="pagination">
+  <!-- first arrow button -->
+  {% if page_format.has_previous %}
+    <li><a href="?page={{ page_format.previous_page_number }}&per_page={{ page_format.per_page }}"aria-hidden="true">&laquo;</a></li>
+  {% else %}
+    <li class="disabled" aria-hidden="true">&laquo;</li>
+  {% endif %}
+  <!-- pagination buttons-->
+  {% if page_format.show_first %}
+    <li><a href="?page=1&per_page={{ page_format.per_page }}" role="button" aria-label="Go to page 1">1</a></li>
+  {% endif %}
+  {% if page_format.first_ellipsis %}
+    <li><span class="ellipsis" aria-hidden='true'>...</span></li>
+  {% endif %}
+
+  {% for linkpage in page_format.page_numbers %}
+    {% ifequal linkpage page_format.page %}
+       <li class="is-active" aria-label="Current page, page {{ page_format.page }}"><b>{{ page_format.page }}</b></li>
+    {% else %}
+       <li><a href="?page={{ linkpage }}&per_page={{ page_format.per_page }}" role="button" aria-label="Go to page {{ linkpage }}">{{ linkpage }}</a></li>
```

Comment:
I don't know if this is a real VO issue, but when I navigate through the options using VO+➡️, it sounds confusing:

"Go to page 7 10 of 11"

<img width="588" alt="vo-issue-maybe" src="https://user-images.githubusercontent.com/3209501/68711846-1aa14f80-0560-11ea-9049-e3a1ed94ccb7.png">

Adding puctuation might help? To be clear, this may be normal VO behavior, it just sounds a bit strange/unclear to my ears. Feel free to update this or not, but wanted to mention

Reply:
adding punctuation